frontend/pages/investor/[id]/index.tsx

```diff
@@ -15,104 +18,128 @@ import LayoutContainer from 'components/layout-container';
   };
-}
-
-const InvestorPage: PageComponent<{}, StaticPageLayoutProps> = (props) => {
-  const intl = useIntl();
-  // const { query } = useRouter();
-  // const { id } = query;
-
-  const aboutInfo: {
-    logo: string;
-    name: string;
-    description: string;
-    text: string;
-    website: string;
-    social: SocialType[];
-    contact: ContactItemType;
-  } = {
-    name: 'NESst',
-    description: 'Non-VC Investment vehicle',
-    text: 'NESsT finances enterprises that generate dignified employment and sustain the planet. Since 1997, we have financed 200 enterprises that have sustained jobs for 70,000 individuals and improved 670,000 lives. We achieve our mission through an Incubator that provides pre-seed capital and business services to improve investment readiness; and the Enterprise Fund that deploys seed-state loans to enterprises seeking capital to scale',
-    logo: '/images/temp-placeholders/nesst-logo.png',
-    website: 'https://www.site.com',
-    social: [
-      { id: 'linked-in', url: 'https://www.linkedin.com' },
-      { id: 'twitter', url: 'https://www.twitter.com' },
-      { id: 'facebook', url: 'https://www.facebook.com' },
-      { id: 'instagram', url: 'https://www.instagram.com' },
-    ],
-    contact: {
-      name: 'User name',
-      email: 'user@example.com',
-      phone: '251912343242342',
-    },
+};
+
+type InvestorPageProps = {
+  investor: Investor;
+  enums: GroupedEnums;
+};
+
+const InvestorPage: PageComponent<InvestorPageProps, StaticPageLayoutProps> = ({
+  investor,
+  enums,
+}) => {
+  const {
+    name,
+    website,
+    twitter,
+    facebook,
+    linkedin,
+    instagram,
+    contact_email,
+    contact_phone,
+    categories,
+    picture,
+    ticket_sizes,
+    instrument_types,
+    impacts,
+    sdgs,
+    about,
+    mission,
+    investor_type,
+    other_information,
+    language,
+    prioritized_projects_description,
+  } = investor;
+  const { category, ticket_size, instrument_type, impact, investor_type: investor_types } = enums;
+
+  const getSocialInfo = () => {
+    const social: SocialType[] = [
+      { id: 'linked-in', url: linkedin },
+      { id: 'twitter', url: twitter },
+      { id: 'facebook', url: facebook },
+      { id: 'instagram', url: instagram },
+    ];
+    return social.filter(({ url }) => !!url);
+  };
+
+  const contact: ContactItemType = {
+    email: contact_email,
+    phone: contact_phone,
+    name,
   };
 
+  const logo = picture.small;
+
   const tagsGridRows: TagsGridRowType[] = [
     {
       title: 'Invests in',
       type: 'category',
-      tags: [
-        { id: 'tourism-and-recreation', name: 'Tourism & Recreation' },
-        { id: 'non-timber-forest-production', name: 'Non-timber forest production' },
-        { id: 'sustainable-agrosystems', name: 'Sustainable agrosystems' },
-        { id: 'forestry-and-agroforestry', name: 'Forestry & agroforestry' },
-      ],
+      tags: category.filter(({ id }) => categories.includes(id)),
```

Comment:
> `categories.includes(id))`

Looking at the schema in Swagger, I believe `categories` is not mandatory. If the endpoint returns `null` this will crash. 
I'd replace this with `categories.includes(id))`.

Same for `impacts` and `instrument_sizes` below (might as well do them all to foolproof it, in case required/non-required fields change in the future).